content/docs/tasks/security/mtls-migration/index.md

```diff
@@ -5,10 +5,22 @@ weight: 80
 keywords: [security,authentication,migration]
 ---
 
-This task shows how to migrate your existing Istio services' traffic from plain text to mutual TLS without breaking live traffic.
-
-In practice, a mesh consists of both Istio services (with Envoy sidecars) and services without Envoy sidecar(called "legacy" below for simplicity).
-A legacy service can't use Istio issued key/certificate to send mutual TLS traffic. We want to enable mutual TLS incrementally, safely.
+This task shows how to migrate your existing Istio services' traffic from plain
+text to mutual TLS without breaking live traffic.
+
+Imagine you have many services talking to each other and you want to
```

Comment:
Imagine you/If you

Reply:
In the scenario where there are many services communicating over the network, it may be desirable to gradually migrate them to Istio.

Reply:
done